server/provisioning/docker/spark-worker/files/default_cmd

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source /root/spark_files/configure_spark.sh
+
+IP=$(ip -o -4 addr list eth0 | perl -n -e 'if (m{inet\s([\d\.]+)\/\d+\s}xms) { print $1 }')
```

Comment:
I agree, but can we do that using just docker/fig? In the future when we have real deployment pipeline then this should definitely be removed.